### ISSUE TITLE
Samsung s6  6.0 does not  work

### ISSUE THREAD

**Issue by dstmath** [Marswin/MarsDaemon#1]

**Comment by Marswin** [comment]
yes,  there are many devices do not suppot still, such as Samsung
6.0,Huawei P6 and some of Xiaomi

2016-03-18 14:45 GMT+08:00 dstmath notifications@github.com:
